api/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "auth",
   "version": "2.0.0",
-  "main": "api/auth.js",
+  "main": "./auth.js",
   "scripts": {
-    "start": "node api/auth.js",
+    "start": "node ./auth.js",
+    "dev": "./config/ensure_key.sh && nodemon",
     "test": "mocha --reporter spec",
-    "apidoc": "apidoc -i api/ -o docs/apidoc/",
+    "apidoc": "apidoc -i . -o docs/apidoc/",
```

Comment:
Some of our API has broken apidoc decleartions and we need to fix it.. just FYI

Reply:
Cool OK! I didn't tested it tbh, just pointed to the dirs